polylogue/index.py

```
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import Any, Dict
```

Ruff (0.15.9)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)